math/minuit/src/TLinearFitter.cxx

```diff
@@ -37,6 +37,8 @@ std::map<TString,TFormula*> TLinearFitter::fgFormulaMap;
 
 \class TLinearFitter
 
+\legacy{TLinearFitter, Consider switching to Minuit2.}
+
```

Comment:
```suggestion
\legacy{TLinearFitter, Consider switching to ROOT::Fit::Fitter, calling the LinearFit() method.}

```
Also here, we should recommend the high-level interface instead of advising to use Minuit2 directly.